### PR TITLE
ethos first commit

### PR DESCRIPTION
**Pull request by iamollas**
Ethos passed all the tests except from this one: 
RUN_SLOW=1 pytest tests/test_dataset_common.py::LocalDatasetTest::test_load_dataset_all_configs_<your-dataset-name>

with this error: 
E               OSError: Cannot find data file. 
E               Original error:
E               [Errno 2] No such file or directory:

### CODE REVIEW

File: datasets/ethos/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# ETHOS Hate Speech Dataset
```
Comment (lhoestq) [inline datasets/ethos/README.md:1]: can you add the yaml tags at the top file the dataset card please ?
You can find more infos here https://github.com/huggingface/datasets/blob/master/ADD_NEW_DATASET.md#tag-the-dataset-and-write-the-dataset-card

Also can you follow the template for the dataset card ? you can leave some sections content with `[More Information Needed]`: https://github.com/huggingface/datasets/blob/master/templates/README.md

File: datasets/ethos/ethos.py [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+                {
+                    "text": datasets.Value("string"),
+                    "violence": datasets.Value("int32"),
+                    "directed_vs_generalized": datasets.Value("int32"),
+                    "gender": datasets.Value("int32"),
+                    "race": datasets.Value("int32"),
+                    "national_origin": datasets.Value("int32"),
+                    "disability": datasets.Value("int32"),
+                    "religion": datasets.Value("int32"),
+                    "sexual_orientation": datasets.Value("int32"),
```
Comment (lhoestq) [inline datasets/ethos/ethos.py:98]: This fields are true or false right ?

Then can you use the ClassLabel feature type here for those fields ?
For example
```
"race": datasets.ClassLabel(names=["false", "true"])
```

You'll need to regenerate the dataset_infos.json file after this change

**Review by lhoestq — COMMENTED** [review]
Nice thanks !

I left a few comments

Also it looks like this PR includes changes about other files than the ones for ethos

Can you create another branch and another PR please ?

File: datasets/ethos/ethos.py (hunk 2 above)
Comment (iamollas) [reply datasets/ethos/ethos.py:98]: Done!

File: datasets/ethos/README.md (hunk 1 above)
Comment (iamollas) [reply datasets/ethos/README.md:1]: @lhoestq Hello! I added the PR1453 addressing all the issues you mentioned 😊! Thanks

**Comment by iamollas** [comment]
> Nice thanks !
> 
> I left a few comments
> 
> Also it looks like this PR includes changes about other files than the ones for ethos
> 
> Can you create another branch and another PR please ?

@lhoestq Should I close this PR?

**Comment by lhoestq** [comment]
You can create another PR and close this one if you don't mind

**Comment by iamollas** [comment]
> You can create another PR and close this one if you don't mind

Perfect! You should see the #1453 PR for the fixed version! Thanks

**Pull request closed without merging**